src/main/resources/project/lib/EC/WebLogic.pm

```diff
@@ -47,6 +61,12 @@ sub generate_exec_path {
     my $wl = shift;
     my $path = $ENV{COMMANDER_WORKSPACE} . '/exec.jython';
 
+    if (parallel_exec_support) {
```

Comment:
I don't like this block. Why don't use if ( parallel_exec_support) { create unique path } else { create fixed path }? This way it looks like a crutch.